.github/workflows/reusable_data_model_gen.yml

```diff
@@ -0,0 +1,67 @@
+name: Reusable Data Model Generator
+
+on:
+  workflow_dispatch:
+
+jobs:
+  vars:
+    name: Set Variables
+    outputs:
+      user: ${{ steps.data.outputs.user }}
+      pass: ${{ steps.data.outputs.pass }}
+    runs-on: ubuntu-22.04
+    timeout-minutes: 1
+    steps:
+      - name: Generate random username and password
+        id: data
+        run: |
+          USERNAME="user$(shuf -i 1000-9999 -n 1)"
+          PASSWORD=$(openssl rand -base64 12)
+
+          echo "user=$USERNAME" >> $GITHUB_OUTPUT
+          echo "pass=$PASSWORD" >> $GITHUB_OUTPUT
+
+  schemaspy:
+    name: Generate Documentation
+    runs-on: ubuntu-22.04
+    needs: [vars]
+    services:
+      postgres:
+        image: postgis/postgis:16-3.4
```

Comment:
Wondering if we just need basic postgres image than postgis image for FAM?

Reply:
yeah I'll change to the same one from our docker, thanks!